src/test/java/net/minecraftforge/debug/PlayerItemPickupEventDebug.java

```diff
@@ -0,0 +1,38 @@
+package net.minecraftforge.debug;
+
+import org.apache.logging.log4j.Logger;
+
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.entity.player.EntityItemPickupEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.Mod.EventHandler;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import net.minecraftforge.fml.common.gameevent.PlayerEvent;
+
+@Mod(modid = PlayerItemPickupEventDebug.MODID, name = PlayerItemPickupEventDebug.NAME, version = PlayerItemPickupEventDebug.VERSION, acceptableRemoteVersions = "*")
+public class PlayerItemPickupEventDebug
+{
+
+    private static final boolean ENABLED = true;
```

Comment:
You should disable this since almost all testers will pick up drops when testing in a world, and the event listener would be very disturbing.

Reply:
I did not know if it should be "enabled" or "disabled" when submitted to the PR.
For example, OnItemUseFirstTest is still enabled, while NoBedSleepingTest does not have an enabled/disabled flag, while ItemFishedTest is disabled. There's no consistency.

Reply:
Then it's fine.

Reply:
LivingDamageEventTest should have a flag to disable it, the log spam is unhelpful.
Generally, event-based test mods should ship disabled by default.

Reply:
@bs2609 I definitely agree that the log spam is unhelpful, just that I could find no indications as to how I should submit it. I'll go ahead and change it to disabled.

Reply:
The test mods that are left enabled by default are generally ones that only work in very specific circumstances, like hopping on a certain block or using a specific item. Anything that might interfere with other tests or anything that creates log messages in normal situations should be disabled.

Reply:
Something something `LivingDamageEventTest`

Reply:
Forge is flawless shh